docs/tables/aws_ecs_task_definition.md

```diff
@@ -0,0 +1,44 @@
+# Table: aws_ecs_task_definition
+
+A task definition is required to run Docker containers in Amazon ECS.
+
+## Examples
+
+### Basic info
```

Comment:
Would recommend to add  network_mode to the query to check what's the  ECS task networking type.